package.json

```diff
@@ -139,7 +139,7 @@
 		"@fluidframework/eslint-config-fluid": "^2.0.0",
```

Comment:
Does build tools (line above this, can't comment on it because of how PRs work...) 0.22.0 have the fix for incremental build you mentioned? I see no update to fluid build in this PR, so I'm confused why its called out in the description of the PR.